lib/seattleflu/id3c/cli/command/etl/redcap_det_scan.py

```diff
@@ -332,7 +332,16 @@ def create_specimen(record: dict, patient_reference: dict) -> tuple:
     # YYYY-MM-DD HH:MM:SS in REDCap
     received_time = record['samp_process_date'].split()[0] if record['samp_process_date'] else None
 
-    note = 'never-tested' if record['able_to_test'] == 'no' else None
+    note = None
+
+    if record['able_to_test'] == 'no':
+        note = 'never-tested'
+    # This is based on values currently in the SCAN REDCap project,
+    # I'm guessing `can-test` was filled in at some point before the answer
+    # code was changed.
+    #   -Jover, 3 April 2020
+    elif record['able_to_test'] in {'yes', 'can-test'}:
```

Comment:
Ah, unrestricted text variables in REDCap...

Reply:
Hrm. Something else is maybe going on here too, which would be good to understand I think. I currently see this distribution of `able_to_test` values as of right now:

![image](https://user-images.githubusercontent.com/79913/78860154-4711a880-79e6-11ea-843d-e50b4590229b.png)


(The last two are the difference between the JSON values `""` and `null` as generated by REDCap. I have no idea if that's a meaningful distinction or not, but for our purposes I think it's the same.)

Ahh! While writing this I just realized. The values I was looking at are the answer _labels_ not the answer _codes_. If I look at codes I do indeed see what Jover saw.

![image](https://user-images.githubusercontent.com/79913/78860128-39f4b980-79e6-11ea-9b56-94d0b28a9d93.png)

![The More You Know](https://media.giphy.com/media/83QtfwKWdmSEo/200w_d.gif)

Reply:
I think maybe this should be an if/else instead of an if/elif though, or at least an if/elif/else, to explicitly handle the "no value" or "no known value" cases. Otherwise the same issue this PR is addressing can occur if a record goes from yes/no → no value.

Reply:
I think I'll go with if/else with the assumption is that a sample can be tested unless explicitly marked "no". (all samples are innocent until proven guilty)